smartsim/_core/launcher/local/local.py

```diff
@@ -41,6 +41,9 @@ def __init__(self) -> None:
         self.task_manager = TaskManager()
         self.step_mapping = StepMapping()
 
+    def add_step_to_mapping_table(self, name: str, step_map: StepMap) -> None:
```

Comment:
Do we need to override this method if it has the same implementation as the `WLMLauncher`?

Reply:
Added implementation to base ABC